### DISCUSSION TITLE
What's the rational behind #3522

### DISCUSSION THREAD

**Discussion by moritzdietz** [woodpecker-ci/woodpecker#3586] [Ask a Question]
I saw the new release and the security enhancement done in #3522. I was wondering where this was discussed and what the rationale behind it is as I couldn't find an issue that is linked to this PR.

Thanks! :)

**Comment by qwerty287** [comment]
It was discussed internally in maintainer chat.

However, it will change again in the next version, see https://github.com/woodpecker-ci/woodpecker/pull/3570.

If you have any issue with this change and the above PR will solve it, please close this discussion and use the PR/linked issue.

**Comment by moritzdietz** [comment]
Perfect, thanks for the pointer! I'll look into the linked PR and issue for further info.
